app/web/components/Select.tsx

```diff
@@ -31,10 +34,16 @@ export default function Select<T extends Record<string | number, string>>({
 }: Omit<SelectProps, "children"> & {
   id: string;
   options: Extract<keyof T, string | number>[];
-  value?: T extends undefined ? string | number : keyof T;
+  value?: T extends undefined
+    ? string | number | number[]
+    : keyof T | Array<keyof T>;
+  menuItems?: boolean;
   optionLabelMap: T;
 }) {
   const classes = useStyles();
+  const OptionComponent = menuItems
+    ? MenuItem
+    : ("option" as unknown as React.ElementType);
```

Comment:
Please do not use `as unknown as React.ElementType`, to me that signals something is wrong with the type and you are coercing it.

Reply:
done ✅